src/CommonITILValidation.php

```diff
@@ -1053,61 +1085,88 @@ public function showSummary(CommonDBTM $item)
+                    'height'          => 120,
+                ]
+            );
+            $validations[$itil_validationstep_id]['edit_link_js']['target'] = "$itil_edit_js_identifier";
+
+            if ($itil_validationstep_id !== $validationstep_id_inloop) {
+                $itil_validationstep = $itil::getValidationStepInstance();
+                if (!$itil_validationstep->getFromDB($itil_validationstep_id)) {
+                    throw new Exception("itil Approval step not found " . $itil_validationstep_id);
+                }
+
+                $validation_step_status = $itil_validationstep->getStatus();
+                $validation_step_achievements = $itil_validationstep->getAchievements();
+
+                $vs = $itil_validationstep;
+                $vs->getFromDB($validation['itils_validationsteps_id']);
+                $validationsteps_id = $vs->fields['validationsteps_id'];
+
+                $validations[$itil_validationstep_id]['validationstep'] = [
+                    'id' => $validation['itils_validationsteps_id'],
+                    'name' => Dropdown::getDropdownName(ValidationStep::getTable(), $validationsteps_id),
+                    // structured to be later replaced by a DTO with Status::isAccepted()|isRefused()|isWaiting() & getStatus()
+                    'status' => [
+                        'waiting' => $validation_step_status === self::WAITING,
+                        'refused' => $validation_step_status === self::REFUSED,
+                        'accepted' => $validation_step_status === self::ACCEPTED,
+                    ],
+                    // structured to be later replaced by a DTO with getAchievement()
+                    'achievement' => [
+                        'waiting' => $validation_step_achievements[self::WAITING],
+                        'refused' => $validation_step_achievements[self::REFUSED],
+                        'accepted' => $validation_step_achievements[self::ACCEPTED],
+                    ],
+                    'minimal_required_validation_percent' => $itil_validationstep->fields['minimal_required_validation_percent'],
+                ];
+
+                $validationstep_id_inloop = $itil_validationstep_id;
+            }
         }
 
         TemplateRenderer::getInstance()->display('components/itilobject/validation.html.twig', [
             'canadd' => $this->can(-1, CREATE, $tmp),
-            'item' => $item,
+            'item' => $itil,
             'itemtype' => static::$itemtype,
             'tID' => $tID,
-            'donestatus' => array_merge($item->getSolvedStatusArray(), $item->getClosedStatusArray()),
+            'donestatus' => array_merge($itil->getSolvedStatusArray(), $itil->getClosedStatusArray()),
             'validation' => $this,
             'rand' => $rand,
             'items_id' => static::$items_id,
+            'sendapproval_modal' => $this->getModalFormHtmlElements('sendapproval_modal', __('Send approval request'), $itil->getID()),
```

Comment:
Why not use the modal Twig macros? More specifically, does the existing `viewsubitem.html.twig` template work for this case (maybe with minor adjustments).

Reply:
At the moment, I won't spend time to do it. It looks not important for me.
But It could be great if you do theses changes. It's probably easy for you, and will make me learn without big efforts :)